bad_code.java

```diff
@@ -0,0 +1,54 @@
+public class BadCodeExample {
+    
+    public static void main(String[] args) {
+        ChildClass child = new ChildClass();
```

Comment:
In the `BadCodeExample` class, consider using a more descriptive name for the `child` variable, such as `myChild`, to improve code readability. [medium]